src/covariance_parameters/utils.jl

```diff
@@ -0,0 +1,42 @@
+"""
+    fix_sign!(A::AbstractMatrix)
```

Comment:
Could this have a more concise name? "Fixing the sign" can be quite a number of things and it's not super clear what this does. I suppose it turns upper triangular square roots into actual Cholesky factors, right?

Reply:
Your understanding is correct. I am not sure what a more descriptive name would be though, ```utrisqrt2utrichol```?

Reply:
I remember that I saw `choleskify` somewhere once, maybe `choleskyfi_squareroot!` could be an option?

Reply:
But I think yours is fine too, if it's only really used in this file it also doesn't matter too much I guess.

Reply:
I went with ```utrisqrt2utrichol```